main.js

```diff
@@ -1,31 +1,36 @@
-//lesson-2
+/* eslint-disable linebreak-style */
+/* eslint-disable func-names */
+/* eslint-disable no-plusplus */
+/* eslint-disable max-len */
+/* eslint-disable linebreak-style */
```

Comment:
lines are duplicated. It's also better to take a closer look at how to fix errors, rather than disable eslint rules)

Reply:
TY. Try to fix it.